printer_mods/ZZP8202/Double_GEAR_ratio_Z/README.md

```diff
@@ -0,0 +1,76 @@
+# Double_GEAR_ratio_Z
+
+## Description: 
+#### Double_GEAR_ratio_Z_belt_assembly_with_spherical_bearing_connection
+
+## Objectives：
+#### Reject Sagging
+##### Enabling the Voron 2.4 Gantry to Defy Gravity
+
+Voron 2.4 uses a belt-driven Z gantry structure. Voron 2.4 theoretically can achieve smaller layer lines compared to most 3D printers that use lead screw structures Due to the linearity of the lead screw affecting the stability of the horizontal position. However, with the original design's 4:1 gear ratio, the static torque of the stepper motor is insufficient to support the heavy gantry in the event of a power outage, especially when mods such as nozzle cleaner and klicky are added to the gantry, further increasing its weight. Since the rear weight of the gantry is significantly greater than the front, the gantry tilt is inevitable, requiring multiple calibrations before each print to achieve an acceptable parallelism with the print bed. To prevent the print head from hitting the platform during leveling, a higher leveling height is also necessary. In my opinion, a bigger issue is that in a power-off state, the gantry and the main frame might stretch, posing unknown deformation risks.
+
+Previously, I modified the original G-code M84 to never turn off the Z motors unless an emergency stop or power cut occurs, to prevent significant gantry tilt.
+
+Additionally, in the current setup, the connection between the gantry and the linear rails is in a 'semi-connected' state, requiring high calibration efforts and even risking disconnection during printing.
+
+In my modification, I used GE5C ball joints to connect the gantry and the linear rails, which should provide a more stable structure. I also used a pulley system to increase the gear ratio to 8:1, allowing the gantry to maintain its position without external forces after a power outage. Practical tests show that only one leveling attempt is needed after a power outage without external influences, compared to the previous 3-4 attempts needed to complete the leveling.
+
+
+## Image:
+![front](Images/CAD_front_view.png)
+![side](Images/CAD_side_view.png)
+![Detal2](Images/CAD_Detal_view2.png)
+![Detal1](Images/CAD_Detal_view1.png)
+![Photo](Images/ezgif.com-resize.gif)
```

Comment:
The README.md is referencing "Images/ezgif.com-resize.gif" but that file doesn't exist